index.bs

```diff
@@ -4902,6 +4991,55 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the authenticator's credential created during a 
```

Comment:
I would suggest: s/the authenticator's credential/the [=public key credential source=]/

Reply:
Agreed - fixed in subsequent commit.